### PR TITLE
Support set for ArrayFields and add test

### PR DESCRIPTION
**Pull request by lindkvis**

### CODE REVIEW

File: src/main/java/org/caffa/rpc/CaffaObjectArrayField.java [hunk 1]
```diff
@@ -24,21 +24,27 @@ public CaffaObjectArrayField(CaffaObject owner, String keyword) {
 
         ArrayList<CaffaObject> objects = new ArrayList<>();
 
-        for (RpcObject object : objectList.getObjectsList())
-        {
+        for (RpcObject object : objectList.getObjectsList()) {
             String jsonString = object.getJson();
             System.out.println("Got child JSON: " + jsonString);
```
Comment (kriben) [inline src/main/java/org/caffa/rpc/CaffaObjectArrayField.java:29]: Log instead of System.out.println?

File: src/test/java/ClientFieldTest.java [hunk 2]
```diff
@@ -117,11 +151,32 @@ void boolVector() {
...
         assertEquals(true, values.get(3));
+
+        {
+            ArrayList<Boolean> newValues = new ArrayList<Boolean>() {
+                {
+                    add(false);
+                    add(false);
+                    add(true);
+                }
+            };
```
Comment (kriben) [inline src/test/java/ClientFieldTest.java:168]: Fancy!

This simpler to understand (and shorter):
ArrayList<Boolean> newValues = new ArrayList<Boolean>();
newValues.add(false);

File: src/main/java/org/caffa/rpc/CaffaFloatArrayField.java [hunk 3]
```diff
@@ -22,19 +29,16 @@ public CaffaFloatArrayField(CaffaObject owner, String keyword) {
...
+        if (this.localArray != null) {
             System.out.print("local");
-        }
-        else{
+        } else {
             System.out.print("grpc");
-        }      
+        }
         System.out.println(" {");
         System.out.println("keyword = " + this.keyword);
```
Comment (kriben) [inline src/main/java/org/caffa/rpc/CaffaFloatArrayField.java:38]: Avoid System.out.println in library code.

**Review by kriben — APPROVED** [review]
Code looks good!

Minor nitpicking about System.out where logging is better.

File: src/main/java/org/caffa/rpc/CaffaObjectArrayField.java (hunk 1 above)
Comment (lindkvis) [reply src/main/java/org/caffa/rpc/CaffaObjectArrayField.java:29]: Agree. I will go over and remove all printouts in a separate commit

**Pull request merged**